jest.config.js

```diff
@@ -11,7 +11,14 @@ module.exports = {
   },
   moduleFileExtensions: ['js', 'json', 'jsx', 'ts', 'tsx', 'node'],
   preset: 'ts-jest',
+  // "resetMocks" resets all mocks, including mocked modules, to jest.fn(),
+  // between each test case.
+  resetMocks: true,
+  // "restoreMocks" restores all mocks created using jest.spyOn to their
```

Comment:
Great description! I will absolutely forget in the future which cases this applies to, so I appreciate the reference to `jest.spyOn` here